### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by scribblemaniac** [czytelny/backlog#68]
Upon starting backlog for the first time on macOS it pops up with an error message and does not proceed any further.

Backlog version: 1.8.0
OS version: macOS 10.13.6

Popup message contents:
```
Uncaught Exception:
Error: ENOENT: no such file or directory, open '/Users/username/Library/Application Support/Backlog/window.json'
    at Object.fs.openSync (fs.js:646:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:172:20)
    at fs.writeFileSync (fs.js:1303:33)
    at e.value (/Applications/Backlog.app/Contents/Resources/app.asar/dist/electron/main.js:1:2609)
    at LodashWrapper.db.read (/Applications/Backlog.app/Contents/Resources/app.asar/node_modules/lowdb/lib/main.js:32:21)
    at module.exports (/Applications/Backlog.app/Contents/Resources/app.asar/node_modules/lowdb/lib/main.js:51:13)
    at /Applications/Backlog.app/Contents/Resources/app.asar/dist/electron/main.js:1:12512
    at Module.<anonymous> (/Applications/Backlog.app/Contents/Resources/app.asar/dist/electron/main.js:1:12792)
    at e (/Applications/Backlog.app/Contents/Resources/app.asar/dist/electron/main.js:1:270)
    at /Applications/Backlog.app/Contents/Resources/app.asar/dist/electron/main.js:1:1078
```

**Comment by czytelny** [comment]
Thanks for bringing this up. It appears on first run. Need to be fixed

**Comment by czytelny** [comment]
@scribblemaniac it would be nice if you can check if this error appers also on 2.0.0 beta version :) it shouldn't: http://backlog.cloud/beta/

**Comment by scribblemaniac** [comment]
@czytelny Thanks I will test it when I get a chance.

**Comment by scribblemaniac** [comment]
Yes, this issue appears to be fixed in the 2.0.0-beta release. The application opens successfully.
